tasks/19.html

```diff
@@ -12,7 +12,35 @@ <h1 class="global-heading">&nbsp;</h1>
 
         <main class="global-results">
             <style>
-                /* your styles here */
+                .global-scope > div
```

Comment:
Кажется, здесь во всех селекторах можно не указывать родительский класс
```suggestion
                div
```